ABOUT.md

```diff
@@ -0,0 +1,51 @@
+
+# What is MineColonies?
+Here, we would like to give you an overview of what our mod is.
+
+### Creativity 
+MineColonies is a town building mod that allows you create your own thriving Colony within Minecraft. It tends to depict a real-life scenario by providing you with different tools in your chest to aid your gaming experience. Featuring many NPC workers such as Builders, Crafters, Farmers, Guards, Enchanters and Alchemists, MineColonies provides the ability to create a rich and unique Colony.
```

Comment:
I would add/write this (but under the previous question "What is Minecolonies?"): "MineColonies is an interactive Town building mod that allows you to create your own thriving Colony within Minecraft. It depicts real-life scenarios by providing you with different options for you to craft in your quest to build your own Colony and enhance your gaming experience. Featuring many NPC workers such as; Builders, Lumberjacks, Farmers, Fishermen, Guards, Barracks, Miners, Smeltery, Baker, Cook/Restaurant, Deliverymen, Animal Herders and more in development. As well a specialized buildings such as; Warehouse, Citizen Huts and a Townhall. MineColonies gives the player the ability to create a Colony as rich and unique as every player."